packages/web/src/App.tsx

```diff
@@ -13,7 +13,7 @@ const App = () => (
       <Route path="/signout" component={SignOut} />
       <Route path="/profile" component={Profile} />
       <Route path="/signup" component={SignUp} />
-      <Route path="/reset" component={PasswordReset} />
+      <Route path="/password-reset" component={PasswordReset} />
```

Comment:
There is a difference between your changes and Vlad's one in feat(13) packages/server/src/constants/pages.ts I'd suggest use one path and page name

Reply:
I will change it to /password-reset on BE constants

Reply:
Nice catch! I actually prefer `/reset` and didn't like changing it, so I'll change it. I just saw email coming with `password-reset`, so that's where it also has to be changed to a shorter `/reset`.

Reply:
So the final decision is' /reset', right?

Reply:
Yes, reset looks better

Reply:
[Fixed](https://github.com/dubbha/croton/pull/38/commits/5836f4f37ab47be4d8fe6ea63ab42c9bf4ade443)